web/ui.js

```diff
@@ -162,15 +162,25 @@ $(function() {
         return "<span class=\"" + (x < 0 ? "neg" : "pos") + "\">" + rnd8(Math.abs(x)) + "</span>";
     }
 
-    function updateGraph() {
+  function getIcon(symbol) {
```

Comment:
There is some indentation issues here but I will be happy to fix these myself post merging!

Reply:
NP.  I also landed a few spot fixes after the fact to fix spacing issues in the UI for missing icons, and also display icons for `SC` and `BLK`...  I suspect we'd need to canonicalize more coin names in the future.  If you had some time to look into what other coins we're consistent in, I'd appreciate that!